modules/fscloud/main.tf

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+module "cbr_zone" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  version          = "1.18.0"
+  name             = var.cbr_zone_name
+  zone_description = "CBR Network zone representing VPC"
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [{
+    type  = "vpc", # to bind a specific vpc to the zone
+    value = var.vpc_crn,
+  }]
+}
+
+module "secrets_manager" {
+  source                           = "../.."
+  resource_group_id                = var.resource_group_id
+  region                           = var.region
+  secrets_manager_name             = var.secrets_manager_name #tfsec:ignore:general-secrets-no-plaintext-exposure
+  sm_service_plan                  = "standard"
+  sm_tags                          = var.sm_tags
+  service_endpoints                = "private"
+  endpoint_type                    = "private"
+  kms_encryption_enabled           = true
+  existing_kms_instance_guid       = var.existing_kms_instance_guid
+  enable_event_notification        = true
```

Comment:
I don't think having event notifications is an fscloud requirement? If not, expose as a variable and make it optional maybe

Reply:
variable